obs-package-status/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM opensuse/tumbleweed
+
+RUN zypper  --non-interactive --quiet ref \
+ && zypper -n in osc ruby2.6 ruby2.6-devel git gcc make autoconf zlib-devel libxml2-devel libxslt-devel
+RUN gem install bundler
+RUN bundle.ruby2.6 config build.nokogiri --use-system-libraries
```

Comment:
Hm this only will write `/root/.bundle/.config`. Are you sure this will have any effect on the subsequent bundler calls?